deployment/ansible/roles/model-my-watershed.rwd/tasks/app.yml

```diff
@@ -3,14 +3,23 @@
   file: path="{{ rwd_data_path }}"
         state=directory
 
-- name: Pull RWD docker container image
-  command: /usr/bin/docker pull {{ rwd_docker_image }}
-
 - name: Configure RWD service definition
-  template: src=upstart-mmw-rwd.conf.j2
-            dest=/etc/init/mmw-rwd.conf
-  notify:
-    - Restart mmw-rwd
+  docker_service:
```

Comment:
Migrated to a Docker managed service vs. wrapping Docker commands in Upstart/`systemd`.